zerver/webhooks/insping/tests.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from typing import Text
+
+from zerver.lib.test_classes import WebhookTestCase
+
+
+class InspingHookTests(WebhookTestCase):
+    STREAM_NAME = 'test'
+    URL_TEMPLATE = "/api/v1/external/insping?&api_key={api_key}"
+    FIXTURE_DIR_NAME = 'insping'
+
+    def test_website_state_available_message(self) -> None:
+        expected_subject = u"insping"
+        expected_message = u"State changed: Available\n" \
+                           u"URL: http://privisus.zulipdev.org:9991\n" \
+                           u"Response time: 223 ms\n" \
+                           u"Timestamp: Fri Dec 29 17:23:46 2017"
+
+        self.send_and_test_stream_message('website_state_available',
+                                          expected_subject, expected_message,
+                                          content_type="application/"
+                                                       "x-www-form-urlencoded")
```

Comment:
This isn't a huge deal but could you please turn this into `content_type="application/x-www-form-urlencoded"`? It looks a bit confusing like this. :)

Reply:
Sure thing, just concerned about the "over indented/under indented" lint thing. I hope it doesn't do that the next time I push these